en/docs/develop/wso2-admin-services.md

```diff
@@ -219,4 +225,4 @@ public class AdminServiceClientManager {
 }
 ```
 
-The complete maven project can be found at: [org.wso2.carbon.sample.admin.service.invoker.zip]({{base_path}}/assets/attachments/develop/org.wso2.carbon.sample.admin.service.invoker.zip)
+For more information, see the complete Maven project - [org.wso2.carbon.sample.admin.service.invoker.zip]({{base_path}}/assets/attachments/develop/org.wso2.carbon.sample.admin.service.invoker.zip)
```

Comment:
I think the earlier one makes more sense because you need the Maven project to build and get the artifact, not to see more information.

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/789/commits/ec4a294f38eaee64b182a0051db6f47677566f0e